wagtail/contrib/typed_table_block/static_src/typed_table_block/scss/typed_table_block.scss

```diff
@@ -1,9 +1,15 @@
 @import 'wagtailadmin/scss/helpers';
 
 .typed-table-block {
+    &__wrapper {
+        overflow-x: auto;
+        // https://stackoverflow.com/questions/6421966/css-overflow-x-visible-and-overflow-y-hidden-causing-scrollbar-issue#answer-39554003
+        padding-bottom: 600px;
+        margin-bottom: -600px;
```

Comment:
Is the purpose of this to hide the horizontal scrollbar when the columns are too wide for the available area? I think seeing the scrollbar is a nice visual cue that that has happened, especially if you weren't the one who added the overflowing column.

The scrollbar is also peeking through the other elements 600px down the page on the page I'm testing with:

![](https://user-images.githubusercontent.com/1044670/137214644-533cc0f7-5f24-47ef-bed8-f7dd02769867.png)


I am also running into this issue reported in the comments below that answer:

> This also blocks pointer events for 250px below the element.

Specifically, the + button to add a block below the TypedTableBlock is very hard to reliably hit with this technique in place.

Reply:
We set ‘overflow-x: auto’ on the wrapper, therefore `overflow-y` becomes `auto` too. Opening up the menu (to choose a new column) will be clipped by the wrapper. To reach the lower menu items the user needs to vertically scroll. 

This hack is to display the menu without scroll. I wasn't aware for the nasty side-effect.

Maybe the best option is to remove this padding-margin-hack and let the users scroll?